### PR TITLE
HTML font preview generation

### PR DESCRIPTION
**Pull request by pguimera**
Duplicated and fixed #46 due to inactivity.

### CODE REVIEW

**Comment by jeerbl** [comment]
Indeed there is an issue with the Travis integration. I'll fix this and come back to you guys.

Thanks,
J

**Comment by pguimera** [comment]
Hey @jeerbl, I fixed the issue with Travis caused by corrupt/outdated lock files. It is now failing because the rules on `semistandard` appear to have changed and it is now complaining about syntax of code that was not updated in this PR.

Shall we apply all of the syntax change suggestions or disable `semistandard` going forward?

**Comment by rndm2** [comment]
@pguimera do you plan to fix this or I can fork and create new one?

**Comment by pguimera** [comment]
Hey @jeerbl and @rndm2, sorry for the delay I've been busy. I've merged master, I hope that solves your issues.

Please let me know if you need any further changes.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "webfonts-loader",
-  "version": "4.2.1",
+  "version": "4.3.1",
```
Comment (jeerbl) [inline package.json]:
```suggestion
  "version": "4.2.1",
```

Please do not update the versioning, I'll take care of that myself when releasing.

Comment (pguimera) [reply package.json]: Fixed, sorry!

**Pull request merged**